### PR TITLE
Fixes for logger

### PR DESCRIPTION
**Pull request by im-kulikov**
- Nop logger (for testing)
- Fix for zap-logger (collides with package name)

### CODE REVIEW

**Review by m1ome — APPROVED** [review]

**Comment by m1ome** [comment]
LGTM

**Comment by im-kulikov** [comment]
@m1ome ok, waiting for @pcherednichenko?

File: logger/zap/zap.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ type Logger struct {
 }
 
 func New() logger.Logger {
-	logger, _ := zap.NewProduction(zap.AddCallerSkip(1))
-	sugar := logger.Sugar()
+	log, _ := zap.NewProduction(zap.AddCallerSkip(1))
```
Comment (pcherednichenko) [inline logger/zap/zap.go:13]: Why skip error here?

Comment (im-kulikov) [reply logger/zap/zap.go:13]: if return an error with logger it will break API

Comment (pcherednichenko) [reply logger/zap/zap.go:13]: I think that basing program should decide what to do with the error, not package, if it is not initialized, then there will be a null pointer error inside the package, which is bad.
And the API just will need to be updated

Comment (m1ome) [reply logger/zap/zap.go:13]: @pcherednichenko what will you do when somehow you cannot init logger? :)

Comment (pcherednichenko) [reply logger/zap/zap.go:13]: @m1ome we can use this logger, for example in serious financial projects, then we should not ignore the error, but in a some funny projects we can

Comment (im-kulikov) [reply logger/zap/zap.go:13]: @m1ome @pcherednichenko I vote for make such changes with error returning in the next version

**Pull request merged**